### DISCUSSION TITLE
Is there a way to disable select for a sprite that is higher depth based on some condition?

### DISCUSSION THREAD

**Discussion by dfireBird** [aevyrie/bevy_mod_picking#303] [Q&A] [answered]
I am making a chess game, in that pieces are in depth 1 and board squares are in depth 0. When I want to move a piece, I first click on the piece and then click on board square to move. 

But since the events only triggers for entity of higher depth, if I click on square with opposite piece, it triggers the click event for the piece. Say if I selected a white piece and I can't move the piece to square with black piece and capture it.

Is there a way based on condition I can temporarily disable click event for some entities in higher depth or make them kinda transparent, such that I can only select the lower depth entities?

**Comment by aevyrie** [comment, marked as answer]
Responded in discord, copying here as well:

> It depends what picking backend you are using whether or not this is supported. I'd look at the drag and drop example, which does this so that when dragging an object it doesn't trigger pointer events while the dragged object is being dragged. 
You could also make the piece a child of the square, that way the event bubbles up to the square even if you click on the piece. 

https://discord.com/channels/691052431525675048/1038322714320052304/1196888893644095619

**Comment by dfireBird** [comment]
I insert the necessary pieces with the `Pickable::IGNORE` used in drag and drop example, which is perfect for my need and works as expected.
